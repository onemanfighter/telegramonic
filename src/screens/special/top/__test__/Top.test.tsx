import { render } from '@testing-library/react';
import Top from '../Top';

describe('Top screen', () => {
  it('should render correctly', () => {
    const { container } = render(<Top />);

    expect(container).toMatchSnapshot();
  });
});
