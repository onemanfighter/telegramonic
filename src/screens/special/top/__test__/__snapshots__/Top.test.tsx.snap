// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top screen should render correctly 1`] = `
<div>
  <div>
    Top
  </div>
</div>
`;
